### DISCUSSION TITLE
KERI implementors call 10/22/2024

### DISCUSSION THREAD

**Discussion by m00sey** [WebOfTrust/keri#101] [Meetings]
**Zoom:**
https://us06web.zoom.us/j/84721071832

**Reports:**
KERIpy - @m00sey (main), @pfeairheller (back up)
SignifyPy - @pfeairheller , @m00sey (back up)
KERIA - @rodolfomiranda , @2byrds (backup)
Signify-TS - @kentbull , @daniellenskjö (backup)
Cesride - @kentbull , @daidoji (backup)
Kerific - @kordwarshuis
kerisse - @henkvancann
keridoc - @henkvancann
signify-browser-extension - @2byrds , @rodolfomiranda (backup)
polaris-web - @2byrds , @rodolfomiranda (backup)

**Comment by m00sey** [comment]
keripy:

- https://github.com/WebOfTrust/keripy/pull/867
- https://github.com/WebOfTrust/keripy/pull/868
- https://github.com/WebOfTrust/keripy/pull/878
- https://github.com/WebOfTrust/keripy/pull/879
- https://github.com/WebOfTrust/keripy/pull/880
- https://github.com/WebOfTrust/keripy/pull/882

**Reply by pfeairheller** [reply]
We will create 1.2.0 release today (10/22)

**Comment by rodolfomiranda** [comment]
## KERIA
### PRs
- https://github.com/WebOfTrust/keria/pull/311
- https://github.com/WebOfTrust/keria/pull/307
- https://github.com/WebOfTrust/keria/pull/304
- https://github.com/WebOfTrust/keria/pull/303
- https://github.com/WebOfTrust/keria/pull/302

### New Issues
- https://github.com/WebOfTrust/keria/issues/312
- https://github.com/WebOfTrust/keria/issues/310
- https://github.com/WebOfTrust/keria/issues/309
- https://github.com/WebOfTrust/keria/issues/308

**Comment by lenkan** [comment]
# Repository WebOfTrust/signify-ts

## Issues created after 2024-10-08

- https://github.com/WebOfTrust/signify-ts/issues/284

## Pull requests opened after 2024-10-08

- https://github.com/WebOfTrust/signify-ts/pull/282
- https://github.com/WebOfTrust/signify-ts/pull/286
- https://github.com/WebOfTrust/signify-ts/pull/289

## Pull requests merged after 2024-10-08

- https://github.com/WebOfTrust/signify-ts/pull/275
- https://github.com/WebOfTrust/signify-ts/pull/280
- https://github.com/WebOfTrust/signify-ts/pull/283
- https://github.com/WebOfTrust/signify-ts/pull/287
- https://github.com/WebOfTrust/signify-ts/pull/288

**Reply by lenkan** [reply]
If people are interested, I have a script that creates this summary.

**Reply by kentbull** [reply]
yes, definitely interested.

**Reply by lenkan** [reply]
https://github.com/lenkan/keri/blob/main/scripts/create-report.sh

The idea was to also have it posting a discussion, but haven't finished that part yet.
